src/site/content/en/blog/coop-coep/index.md

```diff
@@ -28,6 +28,7 @@ feedback:
 
 **Updates**
 
+- **March 5, 2021**: Removed many conditionals required before Chrome 89.
```

Comment:
I suggest being explicit in what's removed from the post. Something like: "Removed limitations for SharedArrayBuffer, debugging functionalities, etc. that are now fully enabled in Chrome 89."
Since there's no archived version of the post, this update is our chance to preserve the history of changes.

Reply:
Makes sense. I will.

Reply:
Fixed. Please check.

Reply:
I wanted to avoid the "etc" and really list all of the removals in this update. I added another suggestion below.